airbyte-integrations/connectors/source-google-analytics-data-api/unit_tests/test_authenticator.py

```diff
@@ -0,0 +1,56 @@
+import datetime
+
+from source_google_analytics_data_api import utils
+from source_google_analytics_data_api.authenticator import GoogleServiceKeyAuthenticator
+
+
+private_key = """
```

Comment:
it looks like a real key, can we replace it with some kind of dummy string?

Reply:
What's wrong with **`it looks like a real key`** ?

Reply:
it makes me think it **is** real

Reply:
Of course it is real. But again, what is wrong with it?

Reply:
Private keys should not be committed to public repositories

Reply:
This private key does not belong to any Airbyte account or any other account in the Universe

Reply:
Only you know it 😉

Reply:
Does it matter if only I know it?

Reply:
well in case this remains in the code and gets merged, I bet I'm not the last person to ask about this key - so the answer is yes

Reply:
I will uncover the mystery of the origin of this private key, or they can solve it by themselves by reading our discussion